Modules/Core/Common/include/itkCommonEnums.h

```diff
@@ -219,7 +219,7 @@ class OctreeEnums
   enum class Octree : uint8_t
   {
     UNKNOWN_PLANE,   ///< The plane is Unknown
-    SAGITAL_PLANE,   ///< The plane is Sagital
+    SAGITAL_PLANE,   ///< The plane is Sagittal
```

Comment:
This suggested that the identifier `SAGITAL_PLANE` might also be misspelled 🤔 Anyway, I think that's beyond the scope of this pull request!

Reply:
Yes, we don't want to bury an interface change in this commit.

Reply:
I created issue #3923 to track it.